VariablesHomework/VariablesHomework.cs

```diff
@@ -60,86 +60,86 @@ public void Operators()
+            Assert.IsTrue(h == 23); // Make this pass
             h--;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 23); // Make this pass
             ++h;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 24); // Make this pass
             --h;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 22); // Make this pass
             h += 1;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 24); // Make this pass
             h -= 1;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 22); // Make this pass
 
             string l = "Hello! ";
             string m = l + "You!";
-            Assert.IsTrue(m == /*Answer here*/); // Make this pass
+            Assert.IsTrue(m == "Hello!You!"); // Make this pass
 
             int v = 5;
             int x = v % 4;
-            Assert.IsTrue(x == /*Answer here*/); // Make this pass
+            Assert.IsTrue(x == 2); // Make this pass
 
             int finalInt = 15;
             int divisionResult = 15 / 3;
-            Assert.IsTrue(divisionResult == /*Answer here*/); // Make this pass
+            Assert.IsTrue(divisionResult == 5); // Make this pass
 
             finalInt = 15;
             finalInt /= 3;
-            Assert.IsTrue(finalInt == /*Answer here*/); // Make this pass
+            Assert.IsTrue(finalInt == 5); // Make this pass
 
             int orderOfOperations = 2;
             orderOfOperations = 2 + 12 * (orderOfOperations * 3);
-            Assert.IsTrue(orderOfOperations == /*Answer here*/); // Make this pass
+            Assert.IsTrue(orderOfOperations == 74); // Make this pass
         }
 
         [TestMethod]
         public void Ranges()
         {
-            Assert.IsTrue(int.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(int.MinValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(float.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(float.MinValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(double.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(double.MinValue == /*Answer here*/); // Make this pass
+            Assert.IsTrue(int.MaxValue == 2147483647); // Make this pass
+            Assert.IsTrue(int.MinValue == -2147483648); // Make this pass
+            Assert.IsTrue(float.MaxValue == 3.4028235E+38f); // Make this pass  //340282346638528859811704183484516925440
+            Assert.IsTrue(float.MinValue == -3.4028235E+38f); // Make this pass
+            Assert.IsTrue(double.MaxValue == 1.7976931348623158E+308); // Make this pass
+            Assert.IsTrue(double.MinValue == -1.7976931348623158E+308); // Make this pass
         }
 
         [TestMethod]
         public void Overflow()
```

Comment:
We haven't yet talked about what happens when you "overflow" outside of a data type's range.